gridsome.server.js

```diff
@@ -36,6 +38,31 @@ class VueI18n {
    * @param {function} param.removePage
    */
   createManagedPages({ findPages, createPage, removePage }) {
+    
+    // Create custom localized routes using the this.options.routes object
+    if (this.options.routes) {
+      this.options.routes.forEach(route => {
+        createPage({
+          path: route.path,
+          component: route.component,
+          context: {
+            locale: route.locale,
+            slug: route.slug
+          },
+          route: {
+            meta: {
+              locale: route.locale
+            }
+          }
+        })
+      });
+
+      return;
+    }
+
+    // Disallow the automatic creation of the localized routes
+    if (!this.options.enablePathGeneration) return;
```

Comment:
maybe I can suggest to directly check for a false value, in order to prevent user to set an incorrect value (like undefined, null) and inadvertently disable the generation.

```suggestion
    if (this.options.enablePathGeneration === false) return;
```